src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,37 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../types/todo';
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users';
+
+interface TodoInfoInt {
+  todo: Todo
+}
+
+interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+}
+
+export const TodoInfo = ({ todo }: TodoInfoInt) => {
+  const getUserById = (id: number): User => {
+    return usersFromServer.find((user) => user.id === id) as User;
```

Comment:
```suggestion
    return usersFromServer.find((user) => user.id === id) || null;
```
it may not find the user, then it will lead to a page crash
it's better to handle the case when result of call `getUserById` returns `null`